cmd/keycloakb/keycloak_bridge.go

```diff
@@ -836,6 +863,63 @@ func config(logger log.Logger) *viper.Viper {
 	return v
 }
 
+func checkMigrationVersion(logger log.Logger, dbParams dbConfig, migrationVersionDBConn CloudtrustDB, minMigrationVersion string) error {
```

Comment:
migrationVersionDBConn seems to be used only locally : you shoud remove it as a parameter and declare it locally

We also can keep the connection open and return it at the end of checkMigrationVersion to avoid to reopen it right after this call in the main function

Reply:
Changed, thanks